src/components/Planner/Planner.tsx

```diff
@@ -42,55 +93,106 @@ const Planner = () => {
         <GridItem colSpan={{ base: 1, md: 8 }} bg="brandGray" p="3">
           <Text>calender and buttons space</Text>
         </GridItem>
-        <GridItem colSpan={{ base: 1, md: 2 }} bg="lightGray">
-          <Center p="3" fontSize="25">
+        <GridItem
+          colSpan={{ base: 1, md: 1 }}
+          bg="lightGray"
+          display="flex"
+          flexDirection="column"
+          justifyContent="center">
+          <Center p="1" fontSize="25">
             <Text>RECIPES</Text>
           </Center>
-          <Droppable droppableId="list">
+        </GridItem>
+
+        {["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"].map(
+          (weekday, index) => (
+            <GridItem
+              key={index}
+              colSpan={{ base: 1, md: 1 }}
+              bg="gray"
+              textAlign="center"
+              p="2"
+              color="brandGray"
+              display="flex"
+              flexDirection="column"
+              justifyContent="center"
+              fontSize="20">
+              {weekday}
+            </GridItem>
+          )
+        )}
+      </Grid>
+      <Grid
+        templateColumns="repeat(8, 1fr)"
+        gap="16px"
+        padding="16px"
+        bg="lightGray">
+        {Object.keys(columns).map((columnId, index) => (
```

Comment:
BLOCKING (although you can handle this in a later PR if you prefer): JavaScript does _not_ guarantee that an object will be ordered. I would consider expanding what you store in each column to include sort order. So, for example:

```
type TColumns = {
   [index: Id]: { sortOrder: number; recipes: Array<TPlannerRecipe>; };
};
```

This will need to be modified when you add the horizontal slots for breakfast/lunch/dinner but should be easy to modify for that.

Reply:
And then you'd want a `.sort` before the `.map` here.

Reply:
Thank you!This makes sense! I will correct it.